roles/validate_operator_bundle/tasks/main.yml

```diff
@@ -26,12 +26,51 @@
 - name: "Set bundle_validate variable"
   set_fact:
     bundle_validate: "{{ operator_work_dir }}"
-  when: not run_upstream|bool
+    when: not run_upstream|bool
```

Comment:
I think this should stay indented as before

Reply:
Thanks, vim often messes up the auto-indentation for ansible